docs/docs/quickstart.rst

```diff
@@ -13,12 +13,12 @@ Configuration
 
 You must provide a custom configuration to set `dbpath` (your database
 location) and `host` (a list of websites for CORS_). All other options have
-sane defaults.
+the same defaults.
```

Comment:
This is incorrect; it does really mean to say "sane", which is different from "the same".